e2e/test/scenarios/question-reproductions/reproductions-3.cy.spec.js

```diff
@@ -1583,48 +1584,42 @@ describe("issue 44668", () => {
   });
 });
 
-// TODO: unskip when metabase#44974 is fixed
-describe.skip(
-  "issue 44974",
-  {
-    tags: ["@external"],
-  },
-  () => {
-    beforeEach(() => {
-      restore("postgres-writable");
-      cy.signInAsAdmin();
-    });
-    it("should not be possible to join with a table or question which is not in the same database (metabase#44974)", () => {
-      startNewQuestion();
-      entityPickerModal().within(() => {
-        entityPickerModalTab("Tables").click();
-        cy.findByText("Sample Database").click();
-        cy.findByText("Orders").click();
-      });
-
-      cy.button("Visualize").click();
-      cy.button("Save").click();
-      modal().button("Save").click();
+describe("issue 44974", () => {
```

Comment:
Let's keep the `.skip` at the highest possible level.
You don't want `beforeEach` executing.